src/viur/core/skeleton.py

```diff
@@ -1422,6 +1425,117 @@ def __txn_delete(skel: SkeletonInstance, key: db.Key):
         for adapter in skel.database_adapters:
             adapter.delete(skel)
 
+    @classmethod
+    def edit(
```

Comment:
I don't like the name. `edit` is already an action in the prototypes. But it has nothing to do with this `edit` method here. To clearly separate them, I suggest calling it [`upsert`](https://en.wiktionary.org/wiki/upsert), as in the context of relational DBs.


```suggestion
    def upsert(
```

Reply:
I don't like the name `upsert` either. Never heard this before.
I see the `edit` in context of *editing* a Skeleton.
You may apply other alternative names.

Reply:
Hmm, okay. Let's brainstorm together in the viur-meeting on Friday. If nobody has a better name, we can merge this as `edit`.

Reply:
What about `Skeleton.apply()`?

Reply:
_apply_ sounds vague in this context. i would expect apply after a sequence of operations to finally apply them to an object.

Further options for our brainstorming / discussion could be:

- change
- modify
- alter

Reply:
Let's name it `patch`?